m4_sessions/1811_inning/failure.md

```diff
@@ -25,6 +29,26 @@
 
 ## Session Outline (Agenda for session, include timing for each session and who is facilitating)
 
+Introductions, name, mod, something you are good at and how long it took you to actually become good at it 5 mins
+
+Short video  talking about how failure is a good thing 5 mins
+
+Redefinition of failure 7 mins. (ask how people would define failure, ask how people feel when they fail, ask what they think of others when they fail)
```

Comment:
Are people journalling?  Are they discussing in small groups?  IN pairs?

Reply:
This will be a large group discussion with calling on people individually